### PR TITLE
Update for schedulers etl

### PR DESCRIPTION
**Pull request by klahnakoski**

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -41,18 +41,18 @@ Fill in the parameters:
...
 
 ```python
 import taskcluster
 options = {
+    "rootUrl": "https://community-tc.services.mozilla.com",
     "credentials": {
-        "accessToken": "nOtalEgItImAtEbAsE64aCCeStOkEnTWWxdUxbvtiI7Q",
-        "clientId": "github/2334429|klahnakoski/test-id"
+        "clientId": "github/2334429|klahnakoski/test-id",
+        "accessToken": "nOtalEgItImAtEbAsE64aCCeStOkEnTWWxdUxbvtiI7Q"
```
Comment (armenzg) [inline scripts/README.md:54]: Are these real values or made up?

File: scripts/etl/schedulers/main.py [hunk 2]
```diff
@@ -9,13 +9,16 @@
 from __future__ import division
 from __future__ import unicode_literals
 
+import logging
```
Comment (armenzg) [inline scripts/etl/schedulers/main.py]: Feel free to land changes in directories under `scripts` without need for review (e.g. scripts/foo/*)

**Review by armenzg — APPROVED** [review]
Not sure if you were asking for review or not.

File: scripts/README.md (hunk 1 above)
Comment (klahnakoski) [reply scripts/README.md:54]: made up

**Pull request closed without merging**